testing/trino-product-tests-launcher/pom.xml

```diff
@@ -132,6 +132,32 @@
             <artifactId>testcontainers</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.confluent</groupId>
+            <artifactId>kafka-protobuf-provider</artifactId>
+            <!-- This is under Confluent Community License and we use it under runtime scope only for tests -->
```

Comment:
looks to be dual-licensed as Apache 2.0 as well now. https://github.com/confluentinc/schema-registry/blob/ef86d4bac917d0798f357b018a8305f0e171ebd8/protobuf-provider/pom.xml#L19.

Same for protobuf-types - https://github.com/confluentinc/schema-registry/blob/ef86d4bac917d0798f357b018a8305f0e171ebd8/protobuf-types/pom.xml#L19

I mention this because we seem to copy these into the final build using `maven-dependency-plugin:copy` (see line 230).

Reply:
nevermind, I see we copy them so that tests can use `File` to get to them.